tools/serverpod_cli/lib/src/create/database_setup.dart

```diff
@@ -53,26 +51,4 @@ class DatabaseSetup {
 
     return true;
   }
-
-  static Future<bool> applyDefaultMigration(Directory dir) async {
-    log.debug('Applying default migration.');
-
-    log.debug('Starting docker container.');
-    var success = await CommandLineTools.startDockerContainer(dir);
```

Comment:
Suggestion: Remove the command from `CommandLineTools`.

These commands are not used by anything anymore right? Please clean them up as well.